vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1282,13 +1316,16 @@ public void deletePictureResource(String uri, ModelCallback callback) {
      * @return a Retrofit response object with the Video as the body
      */
     @Nullable
-    public retrofit2.Response<Video> fetchVideoSync(String uri, CacheControl cacheControl,
-                                                    @Nullable String fieldFilter) {
-        String cacheHeaderValue = createCacheControlString(cacheControl);
+    public retrofit2.Response<Video> getVideoSync(String uri,
+                                                  CacheControl cacheControl,
+                                                  @Nullable String fieldFilter) {
+        final String cacheHeaderValue = createCacheControlString(cacheControl);
         try {
-            return mVimeoService.getVideo(getAuthHeader(), cacheHeaderValue, uri,
-                                          createQueryMap(null, null, fieldFilter)).execute();
-        } catch (IOException e) {
+            return mVimeoService.getVideo(getAuthHeader(),
+                                          uri,
+                                          createQueryMap(null, null, fieldFilter),
```

Comment:
I agree that it is good practice to pull arguments out into a variable and pass them in to keep the call to one line. I will change the call appropriately.